go.mod

```diff
@@ -23,7 +23,7 @@ require (
 )
 
 replace (
-	github.com/gardener/machine-controller-manager => github.com/gardener/machine-controller-manager v0.34.3
+	github.com/gardener/machine-controller-manager => github.com/gardener/machine-controller-manager v0.35.0
```

Comment:
Any specific reason for not updating the `require` section itself? Not sure of the difference in behavior, but only curious.

Reply:
Also, should we change the go-version to go.mod to 1.15 ? Not necessarily in this change, but in general.

Reply:
If we don't specify it explicitly won't it fetch the latest version? Will check it out. 

>Also, should we change the go-version in go.mod to 1.15 ? Not necessarily in this change, but in general.

Yes, let's do that already. Will update this already.